charts/sonarqube-dce/values.yaml

```diff
@@ -116,15 +116,15 @@ ApplicationNodes:
     runAsUser: 1000
 
   readinessProbe:
-    initialDelaySeconds: 120
+    initialDelaySeconds: 180
```

Comment:
don't you think 3 minutes is a little much? this will block SQ from being available for 3 minutes regardless of that actual status. imo we should increase the `failureThreshold`

Reply:
indeed, you are right 👍 I will adjust threshold here